src/main/java/uk/gov/companieshouse/service/dissolution/DissolutionGetter.java

```diff
@@ -41,11 +41,38 @@ public boolean isDirectorPendingApproval(String companyNumber, String officerId)
                 .orElse(false);
     }
 
+    public boolean doesDirectorExist(String companyNumber, String officerId) {
```

Comment:
Quite a bit of code added here. Consider splitting DissolutionDirector classes from Dissolution classes.

Reply:
Can the director-based methods be moved into a separate class? Especially considering that isDirectorPendingApprovalForDissolution and doesDirectorExistInDissolutionRequest don't fetch the dissolution and therefore probs shouldn't be in a Getter class